### DISCUSSION TITLE
ApiKey: Is't possibile to use different apikeys for different controllers?

### DISCUSSION THREAD

**Discussion by biapar** [marcominerva/SimpleAuthentication#129] [Q&A]
ApiKey: Is't possibile to use different apikeys for different controllers?

**Comment by marcominerva** [comment]
You can create a custom implementation of https://github.com/marcominerva/SimpleAuthentication/blob/master/src/SimpleAuthentication.Abstractions/ApiKey/IApiKeyValidator.cs and use the `HttpContext` to get the current Controller, so that you can vslidate your API Key.

**Comment by biapar** [comment]
Thx. Is't possible to use also more  "SchemeName" ?

**Reply by marcominerva** [reply]
No, at this moment SimpleAuthentication supports only a scheme name per Authentication type.
